docs/sysadmin/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+## Troubleshooting playbook
+
+- is it only affecting one user?
+- is it only affecting one sketch / timeline?
+- can the issue be reproduced on demo.timesketch.org?
+- Any output on Chrome console?
+- Try a different device / clear caches from browser
+- Look in nginx / webserver logs
+- Look in Timesketch / celery worker logs
+- Look in ES logs
+- Look in postgres logs
+
+## UX issues
+
+To troubleshoot UX issues, the first look should go to the Chrome developer console and look for any output / errors.
+
+To raise issues related to UX, please provide the following:
+
+- Copy out the full error message(s) from Chrome Console
+- The URL (without your local IP or Domain)
+- What was clicked before it happend
+- Any entries in the server side logs (see below)?
+- Share a screenshot if possible
+- If possible details about the event / sketch
+  - Was it an imported plaso file or CSV or JSONL?
+  - Was the data imported via Web or API client? 
+
+## Docker
+
+To get all your docker instances, run:
+
+```shell
+docker container list
+```
+
+Which will show something like
+
+```shell
+b756f334d281   us-docker.pkg.dev/osdfir-registry/timesketch/dev:latest        "/docker-entrypoint.…"   8 days ago   Up 2 days   127.0.0.1:5000->5000/tcp             timesketch-dev
+7768635b4798   us-docker.pkg.dev/osdfir-registry/timesketch/notebook:latest   "jupyter notebook"       8 days ago   Up 2 days   127.0.0.1:8844->8844/tcp, 8899/tcp   notebook
+6d2e7da9453c   justwatch/elasticsearch_exporter:1.1.0                         "/bin/elasticsearch_…"   8 days ago   Up 2 days   9114/tcp                             es-metrics-exporter
+78443764330b   grafana/grafana:latest                                         "/run.sh"                8 days ago   Up 2 days   127.0.0.1:3000->3000/tcp             grafana
```

Comment:
This is part of the dev container. We have not enabled the metrics on the prod deployment yet. Maybe we should adjust the ouput here in the meantime?

Reply:
yes good point, removed metrics part